lib/fishjam_web/telemetry.ex

```diff
@@ -108,6 +108,25 @@ defmodule FishjamWeb.Telemetry do
         #   - WebSocket traffic
         #   - RTP events (RTSP components don't use ICE)
         #   - HTTP traffic related to metrics (not handled by Phoenix)
+
+        counter("fishjam.rpc_client.call.count",
+          event_name: [:fishjam, :rpc_client, :call],
+          measurement: :duration
+        ),
+        counter("fishjam.rpc_client.multicall.count",
```

Comment:
These metrics should be suffixed `_total`, according to the [Prometheus style guide](https://prometheus.io/docs/practices/naming/)

Additionally, I don't think we need to specify `rpc_client` in the metric name -- it's the internal name of one of our modules, and the `client` part doesn't really communicate anything of value. How do you feel about `fishjam_rpc_call_total` and `fishjam_rpc_multicall_total` instead?